src/components/LayersDataTable.tsx

```diff
@@ -217,16 +238,16 @@ class LayersDataTable extends React.Component {
                             <div className="ui-grid-row">
                                 <div className="ui-grid-col-4" style={{padding:'4px 10px'}}><label htmlFor="srs">Projection</label></div>
                                 <div className="ui-grid-col-8" style={{padding:'4px 10px'}}>
-                                    <InputText id="srs" value={this.selectedLayerType.srs}/>
+                                    <InputText id="srs" value={this.selectedLayerType.srs} readOnly={true}/>
                                 </div>
                             </div>
                             <div className="ui-grid-row">
                                 <div className="ui-grid-col-4" style={{padding:'4px 10px'}}><label htmlFor="id">Bounding Box</label></div>
                                 <div className="ui-grid-col-8" style={{padding:'4px 10px'}}>
-                                    <InputText id="nativeBoundingBoxminX" value={this.selectedLayerType.nativeBoundingBox.minx}/>
-                                    <InputText id="nativeBoundingBoxminX" value={this.selectedLayerType.nativeBoundingBox.maxx}/>
-                                    <InputText id="nativeBoundingBoxmaxY" value={this.selectedLayerType.nativeBoundingBox.miny}/>
-                                    <InputText id="nativeBoundingBoxminY" value={this.selectedLayerType.nativeBoundingBox.maxy}/>
+                                    <InputText id="nativeBoundingBoxminX" value={this.selectedLayerType.nativeBoundingBox.minx} readOnly={true}/>
```

Comment:
map ?
remove

Reply:
I will remove it in the next version. all the table is going to be changed